README.md

```diff
@@ -60,7 +60,27 @@ software. Some key features of this operator are:
 
 ### Custom resources
 
-// TODO: Add a summary of key `ContainerImageScan` (CIS) characteristics
+The Image Scanner operator does currently define a single user-facing Custom
+Resource Definition (CRD), [ContainerImageScan][CIS-CRD] (CIS), that represents the
+Kubernetes API for runtime image scanning of workload container images.
+See [stas_v1alpha1_containerimagescan.yaml][CIS-example] for a (simplified)
+example of a CIS resource.
+
+The CIS resource `.spec` specifies the container image to scan and some
+additional workload metadata, and the image scan result is added/updated
+in `.status` by the `ContainerImageScan` controller.
+
+CIS resources should not be edited by a standard users, as the `Workload`
+controller will create CIS'es from scheduled pods. And the standard Kubernetes
```

Comment:
```suggestion
controller will create CISs from scheduled pods. And the standard Kubernetes
```

I am not quite sure how to write plural form of an acronym like this, but I don't think apostrophe is correct:

> The basic rule is quite simple: **use the apostrophe to indicate possession, not a plural**

Source: https://www.hamilton.edu/academics/centers/writing/seven-sins-of-writing/4

Reply:
```suggestion
controller will create containerimagescans from scheduled pods. And the standard Kubernetes
```

Reply:
Can this bring us further? https://www.port.ac.uk/student-life/help-and-advice/study-skills/written-assignments/acronyms-and-initials-apostrophes-and-plurals

Reply:
Hm, I'm OK with changing to *containerimagescans*, but then we should change it consistently?! And why not camelCase as we use for this resource elsewhere in the description?

Reply:
If we use _CIS resources_ then I think we're good 😄

Reply:
Done.